jgrapht-core/src/main/java/org/jgrapht/graph/AbstractBaseGraph.java

```diff
@@ -516,55 +486,24 @@ public void setEdgeWeight(E e, double weight)
         ((DefaultWeightedEdge) e).weight = weight;
     }
 
-    private Specifics<V, E> createSpecifics()
+    /**
+     * Create the specifics for this graph. Subclasses can override this method in order to adjust
+     * the specifics and thus the space-time tradeoffs of the graph implementation.
+     * 
+     * @return the specifics used by this graph
+     */
+    protected Specifics<V, E> createSpecifics()
```

Comment:
I think this will be another breaking change for the dependencies from Cascading introduced by @cwensel in https://github.com/jgrapht/jgrapht/pull/123.  But I think Chris also told me at some point that moving to JGraphT 1.0 was gonna be a big jump for them anyway due to the Java 8 dependency.  Chris, can you weigh in?